### DISCUSSION TITLE
有人还能联系上法厄同吗？

### DISCUSSION THREAD

**Discussion by Kapler186f** [share121/inter-knot#2630] [其他]
之前听说法厄同被治安局抓了，我还想怎么可能，活跃那么久的传奇绳匠怎么可能说没就没。可最近突然出现个紧急委托需要法厄同出手，却怎么也找不到他，不会真被抓了吧，有能联系上他的麻烦私信我一下，必有重谢。<img src="https://bbs-static.miyoushe.com/static/2024/03/21/eb81560ecf3e308280f8d3599f24b64d_8586799033270022982.png" alt="丽都漫步-艾莲" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/eb81560ecf3e308280f8d3599f24b64d_8586799033270022982.png" alt="丽都漫步-艾莲" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/eb81560ecf3e308280f8d3599f24b64d_8586799033270022982.png" alt="丽都漫步-艾莲" width="75" >

**Comment by Eurus1997** [comment]
看我名字，我是法厄同父亲的同事，V我648把法厄同和伊埃斯拎到你面前。我不知道什么青衣还有<img src="https://bbs-static.miyoushe.com/static/2023/11/16/53069b115197426eff86537576a35d75_2306411376722926481.png" alt="外卖员邦布-出发" width="75" >椒丘，只想帮绳网用户解决问题。

**Reply by Eurus1997** [reply]
我，东风神，打钱

**Comment by yivitte** [comment]
在我印象里，法厄同自7月4日之前都非常活跃，之前想过可能是被治安局抓了，但是那样的传奇绳匠应该不会出这种问题吧

**Comment by Rolesican** [comment]
小心治安局的人在这有卧底（害怕）

**Reply by yivitte** [reply]
哪有猫在耗子的地盘管事的，不用怕

**Comment by WFGoose** [comment]
现在暂时联系不到法厄同，但是我的代理人启明星那里最近来了个新人，听说业务能力很好，可以推荐一下

**Reply by yivitte** [reply]
绳网的新人现在都要么怕苦怕累要么干几天就跑了，能有可靠的新人实在是不容易

**Comment by ddsgkky** [comment]
听说法厄同的接头地点都在六分街，说不定人只是干累了在那边开个店休息一下呢
